modules/ebi-metagenomics/taxonkit/reformat/environment.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: "taxonkit_reformat"
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
```

Comment:
hmmm wondering whether we should remove `defaults` given anaconda gate

Reply:
I would remove it